crates/rspack_plugin_css/src/parser_and_generator/mod.rs

```diff
@@ -33,6 +35,9 @@ use crate::swc_css_compiler::{SwcCssSourceMapGenConfig, SWC_COMPILER};
 use crate::utils::{css_modules_exports_to_string, ModulesTransformConfig};
 use crate::{pxtorem::px_to_rem::px_to_rem, visitors::analyze_dependencies};
 
+static REGEX_IS_MODULES: Lazy<Regex> =
+  Lazy::new(|| Regex::new(r"\.module(s)?\.[^.]+$").expect("Invalid regex"));
```

Comment:
So `module.scss` would be considered as CSS module. Does Webpack do the same thing?

Reply:
with css/auto, yes

Reply:
> So `module.scss` would be considered as CSS module. Does Webpack do the same thing?

Webpack use RegExp in module.rules.test field to specific css files. By default, webpack does not have any rule for scss files, only css default rules in webpack/lib/config/defaults.js.